src/autocomplete/databases/mongo/tests/insert/insert-one.test.ts

```diff
@@ -0,0 +1,16 @@
+import {parseMongoQueryWithoutCursor} from '../..';
+
+test('should not report errors on insertOne statement', () => {
+    const autocompleteResult = parseMongoQueryWithoutCursor(`
+        db.test_collection.insertOne(
+          {
+            test_field: 'test_value'
+          },
+          {
+            test_field: 'test_value'
+          }
+        )
+    `);
```

Comment:
Just to clarify, we can't insert two documents. The second argument has fixed structure
https://www.geeksforgeeks.org/mongodb-insertone-method-db-collection-insertone/

So I'd leave a todo

Reply:
We will need to leave todo in much places if we will look at interfaces, so imo it's not necessary